src/base/utils/fs.cpp

```diff
@@ -518,3 +527,54 @@ QString Utils::Fs::cacheLocation()
         locationDir.mkpath(locationDir.absolutePath());
     return location;
 }
+
+Utils::Fs::FileProcessingGuard::FileProcessingGuard(const QString &path)
+    : m_path{path}
```

Comment:
Then why don't you use the second one? 
Joke.

Old-style initializer has almost full compliance with the constructor call, so we use "function call" appearance.
New-style initializer is used in more contexts, and in some of them it will look ugly and badly readable if you do not add a space. IMO, without space before it looks ugly in all cases (for the most part due to the nature of the appearance of these characters itself).